Solution1/Solution1.Tests/BL/Services/WeatherServiceTests.cs

```diff
@@ -14,22 +21,28 @@ public class WeatherServiceTests
     {
         private readonly IMapper _mapper;
         private readonly WeatherService _weatherService;
+        private readonly IValidator<ForecastWeatherRequestDTO> _validator;
         private readonly Mock<IWeatherApiService> _weatherApiServiceMock;
 
         public WeatherServiceTests()
         {
             _weatherApiServiceMock = new Mock<IWeatherApiService>();
+            _validator = new ForecastWeatherRequestDTOValidator();
             _mapper = new Mapper(MapperConfig.GetConfiguration());
-            _weatherService = new WeatherService(_mapper, _weatherApiServiceMock.Object);
+            _weatherService = new WeatherService(_mapper, _weatherApiServiceMock.Object, _validator);
         }
 
         [Fact]
         public async Task GetByCityNameAsync_ReturnedWeatherDTO_Success()
         {
             // Arrange
             var cityName = "Minsk";
+            var forecast = new ForecastWeatherRequestDTO() { CityName = cityName };
+
```

Comment:
add in style cop rule about empty lines to Error

Reply:
+

Reply:
don't see style cop rule

Reply:
style cop should be added for every project